README.md

```diff
@@ -238,6 +238,7 @@ A few advantages of this solution:
 1. Write code and tests.
 1. Make sure all tests pass `npm test`
 1. Make sure code is well formatted and secure `npm run lint`
+1. Make sure you are using the latest version of both Node.js and npm.
```

Comment:
Why latest? We run workflows with v14 https://github.com/asyncapi/parser-js/blob/master/.github/workflows/if-nodejs-pr-testing.yml#L40 Could you elaborate why you need that?

Reply:
@tinatinz please specify versions like in https://github.com/asyncapi/parser-js/issues/427, so not the latest Node but node>=14 and npm>=7

and I think better to make it step one, of the instruction, so contributors know it upfront

Reply:
> Why latest? We run workflows with v14 https://github.com/asyncapi/parser-js/blob/master/.github/workflows/if-nodejs-pr-testing.yml#L40 Could you elaborate why you need that?

Sorry, I misunderstood #427 and thought the latest version was needed. I've changed the wording and placement like @derberg mentioned.